content/release-10-18-0/digital-twin-manager-10-18-0-bundle/10-18-0-0.md

```diff
@@ -0,0 +1,193 @@
+<tr>
+<td>Users can now move the child assets to another asset which belongs to the same asset model.</td>
+<td>CTM-491</td>
+</tr>
+
+<tr>
+<td>A breadcrumb is shown for the <b>Asset model samples</b> page.</td>
+<td>CTM-563</td>
+</tr>
+
+<tr>
+<td>In the <b>Asset tree</b> page, the count for the number of asset models has been removed.</td>
+<td>CTM-559</td>
+</tr>
+
+<tr>
+<td>In the <b>Asset Tree</b> page, a <b>Reload</b> button is now displayed.</td>
+<td>CTM-555</td>
+</tr>
+
+<tr>
+<td>The import assets layout has been modified to be in sync with other layouts.</td>
+<td>CTM-550</td>
+</tr>
+
+<tr>
+<td>Localizations have been added for the <b>Subassets</b> page.</td>
+<td>CTM-373</td>
+</tr>
+
+<tr>
+<td>Cell renderers have been added for child asset models and asset properties columns in the <b>Export Asset models</b> popup.</td>
+<td>CTM-583</td>
+</tr>
+
+<tr>
+<td>In <b>Asset model</b> creation page, the <b>New asset property</b> button is always enabled and is by default assigned to the asset model after creation.</td>
+<td>CTM-309</td>
+</tr>
+
+<tr>
+<td>A warning message is shown in the <b>Subassets</b> page when the associated asset model or property gets modified.</td>
+<td>CTM-321</td>
+</tr>
+
+<tr>
+<td>A warning message is shown in the <b>Subassets</b> page when the associated asset model is deleted from the backend.</td>
+<td>CTM-333</td>
+</tr>
+
+<tr>
+<td>A warning message is shown in the <b>Subassets</b> page when the associated asset property is deleted from the backend.</td>
+<td>CTM-338</td>
+</tr>
+
+</tbody></table></div>
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>A progress bar is shown during the processing of the file for <b>Import asset model</b>, <b>Import asset properties</b>, and <b>Asset model samples</b>.</td>
+<td>CTM-582</td>
+</tr>
+
+<tr>
+<td>Fixed duplication of asset models when imported from the <b>Asset model samples</b>.</td>
+<td>CTM-578</td>
+</tr>
+
+<tr>
+<td>The export button is disabled when there are no selected properties to be exported.</td>
+<td>CTM-577</td>
+</tr>
+
+<tr>
+<td>Deselecting the child asset models after exporting the asset models is fixed. </td>
+<td>CTM-576</td>
+</tr>
+
+<tr>
+<td>Modified the success messages that are shown while creating, updating, and deleting the <b>Asset properties</b>.</td>
+<td>CTM-562</td>
+</tr>
+
+<tr>
+<td>On clicking the reload button in the <b>Asset tree</b> page, the expanded asset hierarchy is completely closed now.</td>
+<td>CTM-558</td>
+</tr>
+
+<tr>
+<td>Fixed CSS issues in the <b>Asset Models</b> page.</td>
+<td>CTM-554</td>
+</tr>
+
+<tr>
+<td>Child asset models and asset properties are shown on expansion of the parent asset model in the <b>Export Asset Models</b> popup.</td>
+<td>CTM-551</td>
+</tr>
+
+<tr>
+<td>The issues observed in the <b>Asset creation</b> page are fixed.</td>
```

Comment:
Do you mean the page that opens when you want to add a new asset? In this case this entry should be as follows: 

```suggestion
<td>The issues observed in the <b>New asset</b> page are fixed.</td>
```